cms/static/sass/_build.scss

```diff
@@ -61,13 +61,9 @@
 @import 'views/unit';
 @import 'views/container';
 @import 'views/users';
-<<<<<<< HEAD
-@import 'views/checklists';
```

Comment:
Checklists were removed upstream;
Se-won's utilities files caused the conflict.